docs/development/release.md

```diff
@@ -6,13 +6,59 @@ We use a global versioning approach, meaning version numbers in cumulus are cons
 
 Read more about the semantic versioning [here](https://docs.npmjs.com/getting-started/semantic-versioning).
 
+## Pre-release testing
```

Comment:
I feel like we should consider finding a way to automate this, although that's probably dredging up the 'should we integration test the template discussion'.